Tools/Scripts/libraries/webkitbugspy/webkitbugspy/bugzilla.py

```diff
@@ -461,6 +466,66 @@ def add_comment(self, issue, text):
 
         return result
 
+    def issues_to_ids(self, issues):
+        if not isinstance(issues, list):
+            raise TypeError('Input should be a list of Issue objects.')
+        issue_ids = []
+        for i in issues:
+            try:
+                issue_id = i.link
```

Comment:
Not quite sure what the purpose of this check is? Since we actually return the `issue_ids` list, right?